epub32/spec/epub-contentdocs.html

```diff
@@ -853,15 +853,24 @@ <h3>Foreign Resource Restrictions</h3>
 								><code>img</code> element</a>, the following fallback conditions apply to its use:</p>
 
 					<ul>
-						<li>A Core Media Type Resource MUST be specified in its <code>src</code> attribute when it is
-							the child of a <a
-								href="https://www.w3.org/TR/html/semantics-embedded-content.html#the-picture-element"
-									><code>picture</code> element</a>.</li>
-						<li>A Core Media Type Resource MUST be specified in its <code>src</code> attribute when it
-							specifies a <code>srcset</code> attribute.</li>
-						<li>A <a href="epub-packages.html#sec-foreign-restrictions-manifest">manifest fallback</a>
-							[[Packages32]] MUST be specified in all other cases when a Foreign Resource is referenced
-							from its <code>src</code> attribute.</li>
+						<li><p>If it is the child of a <a
+									href="https://www.w3.org/TR/html/semantics-embedded-content.html#the-picture-element"
+										><code>picture</code> element</a>:</p>
+							<ul>
+								<li>it MUST reference Core Media Type Resources from its <code>src</code> and
+										<code>srcset</code> attributes, when those attributes are specified;</li>
+								<li>any child <a
+										href="https://www.w3.org/TR/html/semantics-embedded-content.html#the-source-element"
+											><code>source</code> elements</a> also MUST reference Core Media Type
+									Resources from their <code>src</code> and <code>srcset</code> attributes unless the
+									element specifies that it references a Foreign Resource media type in its
```

Comment:
nitpick: "specifies that it references a Foreign Resource media type in its `type` attribute" is understandable but might sound a little odd or hard to parse?
- we never explicitly specify what is a _Foreign Resource media type_ (is it any media type that is not a CMT? or the media type of a Foreign Resource that is correctly declared in the Package Doc?).
- also, a `source` element can't be said to "reference a (…) media type"
- it doesn't really take into account MIME type parameters, which are allowed in the `type` attribute value.

In HTML/WHATWG lingo, we could say something along the lines of:

> unless its `type` specifies a [MIME type](https://mimesniff.spec.whatwg.org/#mime-type) whose [essence](https://mimesniff.spec.whatwg.org/#mime-type-essence) is listed as a Core Media Type.

Reply:
If we use "essence", everyone has to take a trip to another specification... :)

How about we just keep this in epub parlance by restating it as:

> unless it specifies a media type in its `type` attribute that is not a Core Media Type.

Reply:
> How about we just keep this in epub parlance by restating it as:
> 
> > unless it specifies a media type in its `type` attribute that is not a Core Media Type.

yes, works for me!

> If we use "essence", everyone has to take a trip to another specification... :)

meh, where's the fun when _not_ doing that? 😁🤓